### PR TITLE
[W6.1e][W11-A4] Ng Xin Ler

### PR DESCRIPTION
**Pull request by sherlynng**
LO - JUnit

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +43,30 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception{
```
Comment (joanneong) [inline test/java/seedu/addressbook/common/UtilsTest.java:48]: Could have a spacing between `Exception` and `{`

**Review by joanneong — COMMENTED** [review]
Good attempt made in general :)

Because I didn't have time to share about this in class, but just a reminder that in general, test names follow a 3-part naming rule: `FeatureUnderTest_scenarioBeingTested_expectedOutcome()`, for example: `isAnyNull_nullList_returnsTrue()`.

However, in this case, it's also fine to omit the second and third parts (like you have done) and then comment inside the test case itself :) Note though that this makes the test 'broader', so if one small case fails inside the entire test case will fail, so sometimes people prefer to do the three-part-naming way.

Lastly, if you want to further divide your tests, you can also consider testing cases where the null object is the first object, or somewhere in between, or the last object.
